spec/support/request/web_helper.rb

```diff
@@ -100,8 +100,15 @@ def tomselect_multiselect(value, options)
   def tomselect_search_and_select(value, options)
     tomselect_wrapper = page.find("[name='#{options[:from]}']").sibling(".ts-wrapper")
     tomselect_wrapper.find(".ts-control").click
-    tomselect_wrapper.find(:css, '.ts-dropdown input.dropdown-input').set(value)
+    # Use send_keys as setting the value directly doesn't trigger the search
+    tomselect_wrapper.find(:css, '.ts-dropdown input.dropdown-input').send_keys(value)
+    tomselect_wrapper.find(:css, '.ts-dropdown .ts-dropdown-content .option', text: value).click
+  end
+
+  def tomselect_select(value, options)
+    tomselect_wrapper = page.find("[name='#{options[:from]}']").sibling(".ts-wrapper")
```

Comment:
It may be worth being a bit more explicit with the options:
```suggestion
  def tomselect_select(value, from:)
    tomselect_wrapper = page.find("[name='#{from}']").sibling(".ts-wrapper")
```